Add an optional section for GitHub Copilot Chat (#34)
github/pets-workshop: content/guided-workshop/exercises/5-coding.md

    > **IMPORTANT:** You can ignore any **favicon** errors.

## Bonus – Using GitHub Copilot Chat

> > This section is entirely optional and intended to help you learn more about GitHub Copilot Chat. If this is not of interest, please feel free to progress to the next exercise.

Reviewer: ```suggestion
> **NOTE:** This section is entirely optional and intended to help you learn more about GitHub Copilot Chat. If this is not of interest, please feel free to progress to the next exercise.
```

> In this exercise, we have used GitHub Copilot's Code Completion capabilities to help us stay in the flow and resolve the immediate problem we're trying to solve. Sometimes, we may require a little more assistance from our pair programmer. This is where Copilot Chat can help. Copilot Chat can help us easily document our code, fix issues and help us in our understanding.

Reviewer: ```suggestion
In this exercise, we have used GitHub Copilot's Code Completion capabilities to help us stay in the flow and resolve the immediate problem we're trying to solve. Sometimes, we may want an interactive conversation, where we can explore various paths and provide additional context. This is where Copilot Chat can help. Copilot Chat can help us document our code, fix issues and help us in our understanding.
```

- Removed easily (peeve of mine as it can be a distractor for learners)
- It's more of the style rather than the level of assistance I think for Chat

> There are several ways to interact with Copilot Chat, including the **chat view** and **inline chat**. We'll explore both options.

Reviewer: ```suggestion
There are several ways to interact with Copilot Chat in Visual Studio Code, including the **chat view** and **inline chat**. We'll explore both options.
```

Different IDEs will have different abilities
